helm/oncall/templates/_env.tpl

```diff
@@ -33,10 +33,10 @@
   value: {{ .Values.oncall.slack.clientId | default "" | quote }}
 - name: SLACK_CLIENT_OAUTH_SECRET
   value: {{ .Values.oncall.slack.clientSecret | default "" | quote }}
-- name: SLACK_API_TOKEN
-  value: {{ .Values.oncall.slack.apiToken | default "" | quote }}
-- name: SLACK_API_TOKEN_COMMON
-  value: {{ .Values.oncall.slack.apiTokenCommon | default "" | quote }}
+- name: SLACK_SIGNING_SECRET
+  value: {{ .Values.oncall.slack.signingSecret | default "" | quote }}
+- name: SLACK_INSTALL_RETURN_REDIRECT_HOST
+  value: "https://{{ .Values.base_url | quote }}"
```

Comment:
The `| quote` here will cause in incorrectly formatted value.

Reply:
Well spotted. This was indeed a copy-paste error. I amended the commit with your suggested changes.